packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -273,4 +274,22 @@ export const Query = {
       address,
     }
   },
+  redirect: async (
```

Comment:
@Git-the-Sanz great! I think renaming it to `searchRedirect` would defeat the whole purpose of creating a different query for it. I think the naming is great as it is.

I searched for a match in schema.org and indeed, SearchAction is the closest I could find. Since it offers us no good choices, I think we can leave this as is for now, as it at least is consistent with the search query.

Regarding linen 282, do we have any VTEX official docs regarding this options? If so, I think we should link them in a comment in the code, explaining why there's this restriction.